app/routes/about.tsx

```diff
@@ -1,24 +1,19 @@
-import type { LoaderFunction } from "@remix-run/node";
+import { unstable_defineLoader } from "@remix-run/node";
 import { type MetaFunction, useLoaderData } from "@remix-run/react";
 
 import { PageGrid } from "~/components/PageGrid";
 import { constructSiteTitle } from "~/utils/common";
 
+import team from "../data/team.json";
+
 export const meta: MetaFunction = () => {
 	return [{ title: constructSiteTitle("About") }];
 };
 
-interface LoaderData {
-	members: Record<"name" | "role", string>[];
-	organizersEmeritum: string[];
-}
-
-export const loader: LoaderFunction = async () => {
-	return await import("../data/team.json");
-};
+export const loader = unstable_defineLoader(() => team);
 
 export default function About() {
-	const data = useLoaderData<LoaderData>();
+	const data = useLoaderData<typeof loader>();
```

Comment:
With the import, we get type inference and we can just pass the `loader` and we'll infer from it's return type - no need to manage a `LoaderData` type matching the json structure